### ISSUE TITLE
make check (smoke test)

### ISSUE THREAD

**Issue by ganwell** [adfinis-sygroup/vault-client#46]
In the distro I use we should run "make check" in package build. These checks are usually some kind of smoke test: a simple integration test that proves that the binary is working on the platform. If possible the smoke test should not require spinning up vagrant/docker or use other heavy dependencies.

**Comment by winpat** [comment]
Since `vc` is merly an API client it is rather hard to check for functionality without an existing backend. 

Maybe check the output of `vc -v` to see if the binary can be run? Would that be enough?

**Comment by ganwell** [comment]
Yes. Maybe grep some output, to make sure it does something? Its a bit strange, but the best we can do, right? Would you do a
make check or should it just include that in the package build?
